code/modules/turbolift/turbolift.dm

```diff
@@ -9,14 +9,73 @@
 	var/floor_wait_delay = 85                           // Time to wait at floor stops.
 	var/obj/structure/lift/panel/control_panel_interior // Lift control panel.
 	var/doors_closing = 0								// Whether doors are in the process of closing
+	var/list/music = list('sound/music/elevator.ogg')	// Elevator music to set on areas
```

Comment:
May want to leave this list empty otherwise this code will need to change if the sound file changes/doesn't exist, since it'll fail to compile.